### PR TITLE
Fixes Issue #2604

### PR DESCRIPTION
**Pull request by microweb10**
References
==========
Issue https://github.com/consul/consul/issues/2604

### CODE REVIEW

File: app/views/budgets/investments/_investment_show.html.erb [hunk 1]
```diff
@@ -137,6 +137,15 @@
           <span class="icon-budget"></span>
         </div>
         <br>
+	<% if investment.unfeasible? %>
```
Comment (decabeza) [inline app/views/budgets/investments/_investment_show.html.erb]: Please can you fix the indentation of this file (using two spaces) 🙏 😉 

```
<% if investment.unfeasible? %>
  <div class="callout warning">
    <%= t("budgets.investments.show.project_unfeasible_html") %>
  </div>
<% elsif !investment.selected? %>
  <div class="callout warning">
    <%= t("budgets.investments.show.project_not_selected_html") %>
  </div>
<% end %>
```

Thanks!

Comment (microweb10) [reply app/views/budgets/investments/_investment_show.html.erb]: I have no idea how that happened, I thought I had my emacs already configured for that. Anyway thanks for letting me know. I will have a look into that. I hope now it's OK

Comment (decabeza) [reply app/views/budgets/investments/_investment_show.html.erb]: Don't worry! now looks perfect! 😌

**Comment by decabeza** [comment]
Thanks @microweb10! 😉

**Comment by microweb10** [comment]
Always a pleasure! :)

**Pull request merged**